transport/packet.go

```diff
@@ -91,9 +91,9 @@ type boundPacketConn struct {
 var _ net.Conn = (*boundPacketConn)(nil)
 
 // Dial implements [PacketDialer].Dial.
```

Comment:
[Dial]

Reply:
[[PacketDialer.Dial]. (move the bracket to include the .Dial part, per Hilary

Reply:
interface method names should not be wrapped in `[]`